src/utils/dates.js

```diff
@@ -10,3 +10,12 @@ const ONE_DAY_IN_MILLISECONDS = 86400000;
 export function getFutureDate(offset) {
 	return new Date(Date.now() + offset * ONE_DAY_IN_MILLISECONDS);
 }
+
+export function getDaysBetweenDates(lastTransaction) {
+	let today = new Date();
+	// get both dates in milliseconds
+	let timeDifference = today.getTime() - lastTransaction.toMillis();
+	// change milliseconds into days, rounding up
+	let dayDifference = Math.ceil(timeDifference / ONE_DAY_IN_MILLISECONDS);
```

Comment:
Just a minor suggestion: I was slightly confused when I saw this variable `dayDifference` because in firebase.js we have a variable named `dayDiff`, which semantically sounds very much like the same thing. Should we consider renaming these two variables to be more distinguishable from each other and what they are?

Reply:
appreciate the code comments here